sklearn/preprocessing/_function_transformer.py

```diff
@@ -64,11 +64,31 @@ def fit(self, X, y=None):
             check_array(X, self.accept_sparse)
         return self
 
+    def get_feature_names(self, input_features=None):
+        """
+        Return feature names for output features
+
+        Parameters
+        ----------
+        input_features : list of string, length len(input_features), optional
+            String names for input features if available. By default,
+            "x0", "x1", ... "xn_features" is used.
```

Comment:
oops, you're correct. I was halfway through implementing x0,x1,etc.. when I realized its probably easier to just stay consistent with what you did / what you and @jnothman discussed on the issue page.